Magento_CustomerCustomAttributes/templates/customer/address/edit.phtml

```diff
@@ -123,56 +123,56 @@
                             <?= /* @escapeNotVerified */ __('Street Address %1', $_i) ?>
                         </label>
 
-                        <input type="text" 
-                               name="street[]" 
-                               value="<?= /* @escapeNotVerified */ $block->escapeHtml($block->getStreetLine($_i)) ?>" 
-                               title="<?= /* @escapeNotVerified */ __('Street Address %1', $_i) ?>" 
-                               id="street_<?= /* @escapeNotVerified */ $_i ?>" 
+                        <input type="text"
+                               name="street[]"
+                               value="<?= /* @escapeNotVerified */ $block->escapeHtml($block->getStreetLine($_i)) ?>"
+                               title="<?= /* @escapeNotVerified */ __('Street Address %1', $_i) ?>"
+                               id="street_<?= /* @escapeNotVerified */ $_i ?>"
                                class="input__field <?= /* @escapeNotVerified */ $_streetValidationClass ?>"
                         />
                     </div>
                 <?php endfor; ?>
 
                 <?php if ($this->helper('Magento\Customer\Helper\Address')->isVatAttributeVisible()): ?>
                     <div class="input dashboard__input">
-                        <?php 
+                        <?php
                             $getVatValidationClass = $this->helper('Magento\Customer\Helper\Address')
                                                           ->getAttributeValidationClass('vat_id');
                         ?>
 
                         <label class="label" for="vat_id">
                             <?= /* @escapeNotVerified */ __('VAT Number') ?>
                         </label>
-                        
-                        <input type="text" 
-                               name="vat_id" 
-                               value="<?= /* @escapeNotVerified */ $block->escapeHtml($block->getAddress()->getVatId()) ?>" 
-                               title="<?= /* @escapeNotVerified */ __('VAT Number') ?>" 
-                               class="input__field <?= /* @escapeNotVerified */ $getVatValidationClass ?>" 
+
+                        <input type="text"
+                               name="vat_id"
+                               value="<?= /* @escapeNotVerified */ $block->escapeHtml($block->getAddress()->getVatId()) ?>"
+                               title="<?= /* @escapeNotVerified */ __('VAT Number') ?>"
+                               class="input__field <?= /* @escapeNotVerified */ $getVatValidationClass ?>"
                                id="vat_id"
                         />
                     </div>
                 <?php endif; ?>
 
                 <div class="input dashboard__input required">
-                    <?php 
+                    <?php
                         $getCityValidationClass = $this->helper('Magento\Customer\Helper\Address')
```

Comment:
new var for `helper('Magento\Customer\Helper\Address')` since it used multiple time